src/app/config/no-config/no-config.component.ts

```diff
@@ -10,27 +12,40 @@ import { Config, ConfigService } from '../config.service';
   styleUrls: ['./no-config.component.scss'],
 })
 export class NoConfigComponent implements OnInit {
-  public page = 0;
-  public totalPages = 6;
+  public page: number = 0;
+  public totalPages: number = 7;
 
   public configUpdate: boolean;
   public config: Config;
   public configErrors: string[];
   public configValid: boolean;
   public configSaved: string;
 
-  public octoprintConnection: boolean;
+  public objectvalues = Object.values;
+  public octoprintNodes: any = {
+    'other': {
+      'display': 'Other (> 1.4.0)',
+      'name': 'other',
+      'version': '1.4.0',
+      'url': 'other',
+      'disable': false
+    }
+  };
+  public opInstance: any = this.octoprintNodes['other'];
+  public opApiMsg: string = '';
```

Comment:
I guess we should use octoprintInstance and octoprintApiMessage here, since the full name is used in other vars as well.

Reply:
Sure, I'm lazy I guess ;). I'll update this tonight.